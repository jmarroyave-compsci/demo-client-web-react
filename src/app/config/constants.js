import _package_ from 'package.json';

const Constants = {
	DEBUG : process.env.NEXT_PUBLIC__DEBUG,
	LOCAL : process.env.NEXT_PUBLIC__LOCAL,
	APP : {
		BASE_PATH: process.env.NEXT_PUBLIC__SITE_URL,
		SITE_URL: process.env.NEXT_PUBLIC__SITE_URL,
		VERSION : _package_.version,
		TITLE: process.env.NEXT_PUBLIC__TITLE,
		PAGE_TITLE: process.env.NEXT_PUBLIC__PAGE_TITLE
	},
	SERVERS: {
		DATA: process.env.NEXT_PUBLIC__DATA, 
		GRAPHQL :  process.env.NEXT_PUBLIC__DATA_GRAPHQL,
	},
	DEFAULTS : {
		PAGING_PAGE_SIZE: 10,
	},
	PLUGINS : {
		GOOGLE: {
			ANALYTICS: process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS,
			MAPS: process.env.NEXT_PUBLIC__GOOGLE_MAPS_KEY,
		},
	},
	FEATURES : {
		SEARCH : (process.env.NEXT_PUBLIC__FEATURE_SEARCH == "true"),
	},
}

export default Constants;