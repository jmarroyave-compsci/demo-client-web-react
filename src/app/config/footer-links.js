import _links from './links';

const links = [
  { "name" : "jmarroyave", items: [
    { "name": "blog", "link" : _links.blog },
  ] },
  { "name" : "labs", items: [
    { "name": "stackoverflow", "link" : _links.labs_trends },
    { "name": "mlb", "link" : _links.labs_mlb },
    { "name": "search", "link" : _links.labs_search },
  ] },
  { "name" : "knowledge base", items: [
    { "name": "android", "link" : _links.kb_android },
  ] },
];

export default links;