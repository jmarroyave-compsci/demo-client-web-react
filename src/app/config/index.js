import _package_ from 'package.json';

const Constants = {
	TEST: (process.env.TESTING === "true") ? true : false,
	DEBUG : process.env.NEXT_PUBLIC__DEBUG,
	LOCAL : process.env.NEXT_PUBLIC__LOCAL,
	APP : {
		BASE_PATH: process.env.NEXT_PUBLIC__BASE_PATH,
		SITE_URL: process.env.NEXT_PUBLIC__SITE_URL,
		VERSION : _package_.version,
		TITLE: process.env.NEXT_PUBLIC__TITLE,
		DESCRIPTION: process.env.NEXT_PUBLIC__DESCRIPTION,
		PAGE_TITLE: process.env.NEXT_PUBLIC__PAGE_TITLE,
		HOME_URL: `${process.env.NEXT_PUBLIC__SITE_URL}${process.env.NEXT_PUBLIC__BASE_PATH}`,
	},
	SERVERS: {
		DATA: process.env.NEXT_PUBLIC__SERVER, 
		WEB_SOCKETS: process.env.NEXT_PUBLIC__SERVER, 
		GRAPHQL :  `${process.env.NEXT_PUBLIC__SERVER}/${process.env.NEXT_PUBLIC__SERVER_API_VERSION}/graphql`,
	},
	DEFAULTS : {
		PAGING_PAGE_SIZE: 10,
		IMAGE_PLACEHOLDER: "/img/ph.jpg",
	},
	PLUGINS : {
		GOOGLE: {
			ANALYTICS: process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS,
			MAPS: process.env.NEXT_PUBLIC__GOOGLE_MAPS_KEY,
		},
	},
	FEATURES : {
		SEARCH : (process.env.NEXT_PUBLIC__FEATURE_SEARCH == "true"),
	},
	SOCIAL_NETWORKS: {
		CARD : {
			TITLE : `${process.env.NEXT_PUBLIC__TITLE} v.${_package_.version}`,
			THUMBNAIL_TWITTER: `${process.env.NEXT_PUBLIC__BASE_PATH}${process.env.NEXT_PUBLIC__CARD_THUMBNAIL_TWITTER}`,
		},
		TWITTER: process.env.NEXT_PUBLIC__SOCIAL_NETWORKS_TWITTER,
	}
}

export default Constants;