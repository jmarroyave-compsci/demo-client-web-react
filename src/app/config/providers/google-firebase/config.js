const config = {
  apiKey: "AIzaSyDaNheMMUtsqx8AjiJAWDcfXwAEBYLJUVY",
  authDomain: "labs-f885b.firebaseapp.com",
  projectId: "labs-f885b",
  //projectId: "labs-f885",
  storageBucket: "labs-f885b.appspot.com",
  messagingSenderId: "606766444808",
  appId: "1:606766444808:web:41eec12c129d582727bcbd",
  measurementId: "G-TNBYCW57TS"
};

export default config;