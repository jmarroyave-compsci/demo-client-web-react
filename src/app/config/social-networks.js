const links = [
  /*
	{enable: true, url: 'https://plus.google.com/112618720945759819017', icon: 'google-plus', color: '#d34836'},
	{enable: true, url: 'https://twitter.com/jmarroyave', icon: 'twitter', color: '#00aced'},
	*/
  { enable: true, url: 'https://facebook.com/jmarroyave', icon: 'facebook', color: '#3b5998' },
  { enable: true, url: 'https://www.linkedin.com/in/jmarroyave', icon: 'linkedin', color: '#0077B5' },
  { enable: true, url: 'https://github.com/jmarroyave', icon: 'github', color: '#000' },
  { enable: true, url: 'mailto:jmarroyave.compsci@gmail.com', icon: 'envelope-o', color: '#000' },
];


export default links;