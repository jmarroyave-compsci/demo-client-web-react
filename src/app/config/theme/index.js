import React from 'react';
import { createTheme } from '@mui/material/styles';
import { red } from '@mui/material/colors';

// Create a theme instance.
export const getTheme = ( mode ) => {
  return createTheme({
    palette: {
      mode: mode,
      primary: {
        main: '#556cd6',
      },
      secondary: {
        main: '#19857b',
      },
      error: {
        main: red.A400,
      },
    },
  });
}