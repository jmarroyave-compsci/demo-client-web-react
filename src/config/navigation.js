
class Navigation {
  constructor(){
    this.drawer = {
      show: false,
    };

    this.options = [
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'Home', 
        icon: 'home', 
        route: '/', 
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        separator: true,
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'Movies', 
        icon: 'movie', 
        route: '/movies', 
      },
        {
          showInMenu: false, 
          showInDrawer: false, 
          name: 'Movie', 
          icon: 'movie', 
          route: '/movie/:id/:name', 
        },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'Podcasts', 
        icon: 'podcasts', 
        route: '/podcasts', 
      },
        {
          showInMenu: false, 
          showInDrawer: false, 
          name: 'Podcast', 
          icon: 'podcast', 
          route: '/podcast/:id/:name', 
        },
      {
        showInMenu: true, 
        showInDrawer: true, 
        separator: true,
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'People', 
        icon: 'people', 
        route: '/people', 
      },
        {
          showInMenu: false, 
          showInDrawer: false, 
          name: 'Person', 
          icon: 'person', 
          route: '/person/:id/:name', 
        },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'Awards', 
        route: '/stories/awards/:entity?/:year?/:page?', 
      },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'Stories', 
        route: '/stories', 
      },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'PodcastMusic', 
        route: '/podcasts/stories/music/:page?', 
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        separator: true,
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'About', 
        icon: 'info', 
        route: '/about', 
      },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'Search', 
        icon: 'search', 
        route: '/search/:qry/:page?', 
      },
    ];
  }
}



const nav = new Navigation();
export default nav;