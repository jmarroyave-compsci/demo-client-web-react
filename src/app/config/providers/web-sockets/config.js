import config from 'app/config';

const CONF = {
  SERVER: config.SERVERS.WEB_SOCKETS
};

export default CONF;