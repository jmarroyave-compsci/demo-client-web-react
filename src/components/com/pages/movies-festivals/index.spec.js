import React from 'react';
import {render, fireEvent, waitFor, screen} from '@testing-library/react'
import '@testing-library/jest-dom'

import { itMustHaveNoErrors, TestProvider } from "tests/jest/shared"

import itemState from './automata/tests/item.json'
import TestComponent from './list';
import config from './.config';

describe(`${config.automata.name.toUpperCase()}`, () => {
  beforeEach(() => { 
  }); 
 
  it('compare Page against snapshot', () => {
    const renderer = render(
      <TestProvider state={{[config.automata.name]: itemState}}>
        <TestComponent/>
      </TestProvider>
    )

    itMustHaveNoErrors(renderer)
    expect(renderer.container).toMatchSnapshot();
  });
});
