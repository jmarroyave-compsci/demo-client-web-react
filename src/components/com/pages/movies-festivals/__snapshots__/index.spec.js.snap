// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PAGE_MOVIES_FESTIVALS compare Page against snapshot 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-1 {
  box-sizing: border-box;
  width: 100%;
  height: inherit;
  border-radius: 0;
  background: linear-gradient(60deg, #06388c 75%, #000 150%);
  color: #fff;
}

@media (min-width:0px) {
  .emotion-1 {
    padding: 1.5rem 1.5rem 1.5rem 1.5rem;
  }
}

@media (min-width:600px) {
  .emotion-1 {
    padding: 2rem 3rem 2rem 3rem;
  }
}

@media (min-width:900px) {
  .emotion-1 {
    padding: 3rem 2.5rem 3rem 2.5rem;
  }
}

.emotion-3 {
  padding: 0 0 1rem 0;
  margin: 0;
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  font-size: 1rem;
}

@media (min-width:0px) {
  .emotion-4 {
    padding: 0rem 2rem 3rem 2rem;
  }
}

@media (min-width:600px) {
  .emotion-4 {
    padding: 0rem 2rem 3rem 2rem;
  }
}

@media (min-width:1200px) {
  .emotion-4 {
    padding: 0rem 4rem 3rem 4rem;
  }
}

.emotion-5 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  width: 100%;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-6 {
  box-sizing: border-box;
  margin: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-basis: 100%;
  -ms-flex-preferred-size: 100%;
  flex-basis: 100%;
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  max-width: 100%;
}

@media (min-width:600px) {
  .emotion-6 {
    -webkit-flex-basis: 100%;
    -ms-flex-preferred-size: 100%;
    flex-basis: 100%;
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    max-width: 100%;
  }
}

@media (min-width:900px) {
  .emotion-6 {
    -webkit-flex-basis: 75%;
    -ms-flex-preferred-size: 75%;
    flex-basis: 75%;
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    max-width: 75%;
  }
}

.emotion-7 {
  padding: 0rem 0rem 4rem 0rem;
}

.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-8>:not(style)+:not(style) {
  margin: 0;
  margin-top: 16px;
}

.emotion-9 {
  padding: 0;
  margin: 1rem 0 0.5rem 0;
  text-transform: lowercase;
  font-size: 2.5rem;
}

.emotion-11 {
  box-sizing: border-box;
  margin: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-basis: 100%;
  -ms-flex-preferred-size: 100%;
  flex-basis: 100%;
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  max-width: 100%;
}

@media (min-width:600px) {
  .emotion-11 {
    -webkit-flex-basis: 50%;
    -ms-flex-preferred-size: 50%;
    flex-basis: 50%;
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    max-width: 50%;
  }
}

@media (min-width:900px) {
  .emotion-11 {
    -webkit-flex-basis: 50%;
    -ms-flex-preferred-size: 50%;
    flex-basis: 50%;
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    max-width: 50%;
  }
}

.emotion-12 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  overflow: hidden;
  display: inline-block;
  text-align: center;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  font-size: 1.5rem;
}

.emotion-16 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  text-align: center;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  font-size: 1.5rem;
  padding: 8px;
  border-radius: 50%;
  overflow: visible;
  color: rgba(0, 0, 0, 0.54);
  -webkit-transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  color: #06388c;
}

.emotion-16::-moz-focus-inner {
  border-style: none;
}

.emotion-16.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-16 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-16:hover {
  background-color: rgba(0, 0, 0, 0.04);
}

@media (hover: none) {
  .emotion-16:hover {
    background-color: transparent;
  }
}

.emotion-16:hover {
  background-color: rgba(6, 56, 140, 0.04);
}

@media (hover: none) {
  .emotion-16:hover {
    background-color: transparent;
  }
}

.emotion-16.Mui-disabled {
  background-color: transparent;
  color: rgba(0, 0, 0, 0.26);
}

.emotion-20 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-42 {
  box-sizing: border-box;
  margin: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-basis: 100%;
  -ms-flex-preferred-size: 100%;
  flex-basis: 100%;
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  max-width: 100%;
}

@media (min-width:600px) {
  .emotion-42 {
    -webkit-flex-basis: 100%;
    -ms-flex-preferred-size: 100%;
    flex-basis: 100%;
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    max-width: 100%;
  }
}

@media (min-width:900px) {
  .emotion-42 {
    -webkit-flex-basis: 25%;
    -ms-flex-preferred-size: 25%;
    flex-basis: 25%;
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    max-width: 25%;
  }
}

.emotion-43 {
  background-color: #fff;
  color: rgba(0, 0, 0, 0.87);
  -webkit-transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border-radius: 4px;
  box-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);
}

.emotion-44 {
  color: rgba(0, 0, 0, 0.87);
  background-color: #fff;
  padding: 1rem;
}

.emotion-45 {
  font-size: 1.5rem;
  padding-bottom: 0.5rem;
  color: inherit;
}

.emotion-46 {
  font-size: 0.9rem;
  color: inherit;
}

.emotion-47 {
  cursor: pointer;
  color: inherit;
  background-color: transparent;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  text-decoration-style: dotted;
  text-decoration-color: inherit;
  text-decoration-thickness: 1px;
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
  display: inline-block;
  width: 100%;
}

<div>
  <div
    class="emotion-0"
  >
    <div
      style="margin-bottom: 2rem;"
    >
      <div
        class="com-layout-section section-0 MuiBox-root emotion-1"
      >
        <div
          class="emotion-0"
        >
          <h2
            class="emotion-3"
          >
            Movie Festivals
          </h2>
          
        </div>
      </div>
    </div>
    <div
      class="emotion-4"
    >
      <div
        class="MuiGrid-root MuiGrid-container page-module emotion-5"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-9 emotion-6"
        >
          <div
            class="emotion-7"
          >
            <div
              class="emotion-8"
            >
              <h2
                class="emotion-9"
              >
                general
              </h2>
              <div
                class="MuiGrid-root MuiGrid-container emotion-5"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6 emotion-11"
                  style="padding-right: 1rem;"
                >
                  <div
                    class="makeStyles-card-9 makeStyles-cardHeader-4"
                  >
                    <div
                      class="makeStyles-cardHeader-13 makeStyles-primaryCardHeader-21 makeStyles-cardHeaderStats-15 makeStyles-cardHeaderIcon-16"
                    >
                      <div
                        class="makeStyles-cardIcon-23 makeStyles-primaryCardHeader-28"
                      >
                        <span
                          aria-hidden="true"
                          class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                        >
                          festival
                        </span>
                      </div>
                      <p
                        class="makeStyles-cardCategory-5"
                      >
                        movie festivals
                      </p>
                      <h3
                        class="makeStyles-cardTitle-7"
                      >
                        <div
                          style="z-index: 9999; position: absolute; top: 0px; right: 0px; left: 0px; bottom: 0px; background-color: rgba(0, 0, 0, 0.376);"
                        />
                      </h3>
                    </div>
                  </div>
                </div>
              </div>
              <h2
                class="emotion-9"
              >
                segments
              </h2>
              <div
                class="MuiGrid-root MuiGrid-container emotion-5"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6 emotion-11"
                  style="padding-right: 1rem;"
                >
                  <div
                    class="makeStyles-card-9"
                    style="display: flex; flex-flow: column;"
                  >
                    <div
                      class="makeStyles-cardHeader-13 makeStyles-primaryCardHeader-21"
                    >
                      <h4
                        class="makeStyles-cardTitleWhite-31"
                      >
                        started
                      </h4>
                      <p
                        class="makeStyles-cardCategoryWhite-30"
                      />
                    </div>
                    <div
                      class="makeStyles-cardBody-32"
                      style="flex: 1;"
                    >
                      <div
                        style="z-index: 9999; position: absolute; top: 0px; right: 0px; left: 0px; bottom: 0px; background-color: rgba(0, 0, 0, 0.376);"
                      />
                    </div>
                    <div
                      class="makeStyles-cardFooter-35 makeStyles-cardFooterStats-38"
                    >
                      <div
                        style="padding: 0.5rem;"
                      >
                        <button
                          class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorPrimary MuiIconButton-sizeMedium emotion-16"
                          disabled=""
                          style="padding: 0.15rem 0.25rem;"
                          tabindex="-1"
                          type="button"
                          variant="outlined"
                        >
                          <span
                            aria-hidden="true"
                            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                            icon="bar_chart"
                          >
                            bar_chart
                          </span>
                        </button>
                        <button
                          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorPrimary MuiIconButton-sizeMedium emotion-16"
                          style="padding: 0.15rem 0.25rem;"
                          tabindex="0"
                          type="button"
                          variant="outlined"
                        >
                          <span
                            aria-hidden="true"
                            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                            icon="table_chart"
                          >
                            table_chart
                          </span>
                          <span
                            class="MuiTouchRipple-root emotion-20"
                          />
                        </button>
                        <button
                          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorPrimary MuiIconButton-sizeMedium emotion-16"
                          style="padding: 0.15rem 0.25rem;"
                          tabindex="0"
                          type="button"
                          variant="outlined"
                        >
                          <span
                            aria-hidden="true"
                            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                            icon="table_rows"
                          >
                            table_rows
                          </span>
                          <span
                            class="MuiTouchRipple-root emotion-20"
                          />
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6 emotion-11"
                  style="padding-right: 1rem;"
                >
                  <div
                    class="makeStyles-card-9"
                    style="display: flex; flex-flow: column;"
                  >
                    <div
                      class="makeStyles-cardHeader-13 makeStyles-primaryCardHeader-21"
                    >
                      <h4
                        class="makeStyles-cardTitleWhite-31"
                      >
                        country
                      </h4>
                      <p
                        class="makeStyles-cardCategoryWhite-30"
                      />
                    </div>
                    <div
                      class="makeStyles-cardBody-32"
                      style="flex: 1;"
                    >
                      <div
                        style="z-index: 9999; position: absolute; top: 0px; right: 0px; left: 0px; bottom: 0px; background-color: rgba(0, 0, 0, 0.376);"
                      />
                    </div>
                    <div
                      class="makeStyles-cardFooter-35 makeStyles-cardFooterStats-38"
                    >
                      <div
                        style="padding: 0.5rem;"
                      >
                        <button
                          class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorPrimary MuiIconButton-sizeMedium emotion-16"
                          disabled=""
                          style="padding: 0.15rem 0.25rem;"
                          tabindex="-1"
                          type="button"
                          variant="outlined"
                        >
                          <span
                            aria-hidden="true"
                            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                            icon="bar_chart"
                          >
                            bar_chart
                          </span>
                        </button>
                        <button
                          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorPrimary MuiIconButton-sizeMedium emotion-16"
                          style="padding: 0.15rem 0.25rem;"
                          tabindex="0"
                          type="button"
                          variant="outlined"
                        >
                          <span
                            aria-hidden="true"
                            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                            icon="table_chart"
                          >
                            table_chart
                          </span>
                          <span
                            class="MuiTouchRipple-root emotion-20"
                          />
                        </button>
                        <button
                          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorPrimary MuiIconButton-sizeMedium emotion-16"
                          style="padding: 0.15rem 0.25rem;"
                          tabindex="0"
                          type="button"
                          variant="outlined"
                        >
                          <span
                            aria-hidden="true"
                            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                            icon="table_rows"
                          >
                            table_rows
                          </span>
                          <span
                            class="MuiTouchRipple-root emotion-20"
                          />
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6 emotion-11"
                  style="padding-right: 1rem;"
                >
                  <div
                    class="makeStyles-card-9"
                    style="display: flex; flex-flow: column;"
                  >
                    <div
                      class="makeStyles-cardHeader-13 makeStyles-primaryCardHeader-21"
                    >
                      <h4
                        class="makeStyles-cardTitleWhite-31"
                      >
                        continent
                      </h4>
                      <p
                        class="makeStyles-cardCategoryWhite-30"
                      />
                    </div>
                    <div
                      class="makeStyles-cardBody-32"
                      style="flex: 1;"
                    >
                      <div
                        style="z-index: 9999; position: absolute; top: 0px; right: 0px; left: 0px; bottom: 0px; background-color: rgba(0, 0, 0, 0.376);"
                      />
                    </div>
                    <div
                      class="makeStyles-cardFooter-35 makeStyles-cardFooterStats-38"
                    >
                      <div
                        style="padding: 0.5rem;"
                      >
                        <button
                          class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorPrimary MuiIconButton-sizeMedium emotion-16"
                          disabled=""
                          style="padding: 0.15rem 0.25rem;"
                          tabindex="-1"
                          type="button"
                          variant="outlined"
                        >
                          <span
                            aria-hidden="true"
                            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                            icon="bar_chart"
                          >
                            bar_chart
                          </span>
                        </button>
                        <button
                          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorPrimary MuiIconButton-sizeMedium emotion-16"
                          style="padding: 0.15rem 0.25rem;"
                          tabindex="0"
                          type="button"
                          variant="outlined"
                        >
                          <span
                            aria-hidden="true"
                            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                            icon="table_chart"
                          >
                            table_chart
                          </span>
                          <span
                            class="MuiTouchRipple-root emotion-20"
                          />
                        </button>
                        <button
                          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorPrimary MuiIconButton-sizeMedium emotion-16"
                          style="padding: 0.15rem 0.25rem;"
                          tabindex="0"
                          type="button"
                          variant="outlined"
                        >
                          <span
                            aria-hidden="true"
                            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium emotion-12"
                            icon="table_rows"
                          >
                            table_rows
                          </span>
                          <span
                            class="MuiTouchRipple-root emotion-20"
                          />
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-3 emotion-42"
        >
          <div
            style="padding-left: 1rem; padding-right: 0.5rem;"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 emotion-43"
            >
              <div
                class="emotion-44"
              >
                <div
                  class="emotion-45"
                >
                  See also
                </div>
                <div
                  class="emotion-46"
                >
                  <a
                    href="/about/log"
                    style="flex: 1; display: flex; width: 100%;"
                  >
                    <span
                      class="emotion-47"
                    >
                      Project's log
                    </span>
                  </a>
                </div>
                <div
                  class="emotion-46"
                >
                  <a
                    href="/about/timeline"
                    style="flex: 1; display: flex; width: 100%;"
                  >
                    <span
                      class="emotion-47"
                    >
                      Project's timeline
                    </span>
                  </a>
                </div>
                <div
                  class="emotion-46"
                >
                  <a
                    href="/podcasts"
                    style="flex: 1; display: flex; width: 100%;"
                  >
                    <span
                      class="emotion-47"
                    >
                      Podcasts Dashboard
                    </span>
                  </a>
                </div>
                <div
                  class="emotion-46"
                >
                  <a
                    href="/movies/stories/topics"
                    style="flex: 1; display: flex; width: 100%;"
                  >
                    <span
                      class="emotion-47"
                    >
                      Movie topics
                    </span>
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
